src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Machine> get() {
```

Comment:
I think ExcavatorProducer should return a list of Excavators, doesn't it? 

Otherwise, a possible situation that ExcavatorProducer#get may return a Bulldozer, that isn't expected logic.